pilot/pkg/bootstrap/server.go

```diff
@@ -472,6 +469,60 @@ func (s *Server) initMCPConfigController(args *PilotArgs) error {
 		conns = append(conns, conn)
 	}
 
+	// TODO: remove the below branch when `--mcpServerAddrs` removed
```

Comment:
@Nino-K and other CF folks, is this okay with you? Defining a config file (yaml) from which Pilot would retrieve these values?
@hzxuzhonghu make sure that the config file can be accessed in environments where there is no kubernetes. i.e. we should be able to do `./pilot --config <configFileName>` . I think we already do that.

Reply:
`--meshConfig` is the flag can be used now.